src/Core/DevOps/StaticAnalyze/PHPStan/Rules/PackageAnnotationRule.php

```diff
@@ -118,34 +39,17 @@ public function processNode(Node $node, Scope $scope): array
             return [];
         }
 
-        $area = $this->getProductArea($node);
-
         if ($this->hasPackageAnnotation($node)) {
             return [];
```

Comment:
Should we check if the domain is actually a valid one? So we would see all old area's and mistyped domains immediately 😅

Reply:
At least I'm trying to experiment with that idea here in the ESLint Rule:
https://github.com/shopware/shopware/pull/6334

Reply:
we are already doing that with a phpstan type, no need for  a custom rule https://github.com/shopware/shopware/blob/14370aceb0a5fb70f1c867ae3ade9580e94cf50f/src/Core/Framework/Log/Package.php#L8